docs/changes.rst

```diff
@@ -14,6 +14,7 @@ Not yet released.
 * Improved performance of translation, component or project removal.
 * Improved bulk edit performance.
 * Fixed preserving needs editing and approved states for ODF files.
+* Improved download customization interface.
```

Comment:
```suggestion
* Improved interface for downloading customisations.
```
Or does it mean: "Improved interface for customising downloads"?

Reply:
Improved interface for download customizations.